listenbrainz/webserver/static/js/src/listens/ListenControl.tsx

```diff
@@ -28,13 +29,14 @@ const ListenControl = (props: ListenControlProps) => {
     iconClassName,
     action,
     icon,
-    title,
+    text,
     dataToggle,
     dataTarget,
     disabled,
     link,
     anchorTagAttributes,
     ariaLabel,
+    title,
```

Comment:
Considering how the component is used in ListenCard, where we duplicate title and text, how do you feel about defaulting the `title` to the value of `text`?
Something like this maybe?

`const titleAttribute = title ?? text;`

That way if you pass a text and are OK with the title being the same (which will probably happen most of the time) there's no need to pass a separate prop with the same content.

It would be worth adding a comment on the prop definition in ListenControlProps just to say it defaults to `text` if omitted.

Reply:
@MonkeyDo exactly! I had the same thought regarding this, since we took a very similar approach for `aria-label` in the same component. I had planned to ask you, but guess it slipped out of my mind!

Also, the `title` at line 48 was a big oopsie, my bad. Thanks a lot again.